CMakeLists.txt

```diff
@@ -54,16 +54,13 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY_RELEASE "${LIBRARY_OUTPUT_PATH}")
 # Some non-standard CMake modules
 list(APPEND CMAKE_PREFIX_PATH ${PROJECT_SOURCE_DIR}/configure/cmake)
 list(APPEND CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/configure/cmake)
-include(FindPythonModule)
-include(Utils)
-
-message(STATUS "CMAKE_PREFIX_PATH=${CMAKE_PREFIX_PATH}")
+INCLUDE(Utils)
```

Comment:
Why is this `include` uppercase?

Reply:
It's not - I think Github is showing an incorrect diff in its UI (it happened a couple times before, not sure why..)

Reply:
I just discovered that `cmake-format` does standardize these cases as well, so that's great.